share/spice/sublime_packages/sublime_packages.js

```diff
@@ -0,0 +1,55 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_sublime_packages = function(api_result){
+
+        if (!api_result || api_result.error || api_result.packages.length == 0) {
+            return Spice.failed('sublime_packages');
+        }
+
+        DDG.require('moment.js', function() {
+                Spice.add({
+                    id: "sublime_packages",
+                    name: "Software",
+                    data: api_result.packages,
+                    meta: {
+                        itemType: "Sublime Text Packages",
+                        sourceName: "packagecontrol.io",
+                        sourceUrl: 'http://packagecontrol.io' + api_result.name
```

Comment:
Looks like `api_result.name` can be `undefined`:

![screen shot 2015-04-08 at 12 54 24 am](https://cloud.githubusercontent.com/assets/81969/7039071/e3858e6a-dd89-11e4-9e1a-d9e28bc39982.png)